.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
[question] Where do we stand on copyright (missing here) and license comments (missing in other yml etc) when it comes to these auxiliary files?

Reply:
ultimately we need to go add them all..... to keep the ASWF scanner happy :)

Reply:
So is that a [plan]?

Reply:
I added them for the other yaml for consistency [here](https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/3/commits/71fb9312b8afe6dc9d3c95f3744ed32f0a7f6e9a), but yeah, we need one for all the repos, including work out what we want to do about `.md`

Reply:
Added https://github.com/OpenAssetIO/OpenAssetIO/issues/719